tracer/src/Datadog.Trace/Generated/net461/Datadog.Trace.SourceGenerators/Datadog.Trace.SourceGenerators.TagsListGenerator.TagListGenerator/GrpcServerTags.g.cs

```diff
@@ -0,0 +1,55 @@
+﻿// <auto-generated/>
+#nullable enable
+
+using Datadog.Trace.Processors;
+
+namespace Datadog.Trace.Tagging
+{
+    partial class GrpcServerTags
```

Comment:
I'm a little confused. Is this file supposed to be generated? If so, should there be a generated partial class for `GrpcClientTags`?

Reply:
Ah, nice catch! This is an artifact of my removal of the `language` tag in https://github.com/DataDog/dd-trace-dotnet/pull/2535/commits/ab6e2a7801f2c0779a13e887edabe4893a6bf7bb 🙂 Previously, the `GrpcServerTags.g.cs` _was_ required, because it had an extra property, while `GrpcClientTags` didn't (so wasn't generated). But after removing that from `GrpcServerTags`, the `GrpcServerTags.g.cs` file was no longer generated. 

_However_, I had already committed the file to source control. The compiler doesn't _delete_ files persisted to disk, so this was hanging around. I think this is a failure of our "Verify source generator changes" GitHub Action Workflow so I'll update that to catch this issue. In the meantime, I've removed the extra files in https://github.com/DataDog/dd-trace-dotnet/pull/2535/commits/5290191602dd6e0bd6e312835b9b8c85e662012f

Reply:
Added support for detecting this scenario in https://github.com/DataDog/dd-trace-dotnet/pull/2614